nvflare/private/fed/client/communicator.py

```diff
@@ -87,27 +87,32 @@ def __init__(
 
         self.logger = logging.getLogger(self.__class__.__name__)
 
-    def client_registration(self, client_name, servers, project_name):
+    def client_registration(self, client_name, project_name, register_data: dict, fl_ctx: FLContext):
```

Comment:
Don't think "register_data" is needed since you are sending public props in fl_ctx to peers via peer_ctx already.

Reply:
already removed.